code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -712,7 +718,8 @@
 	S.visible_message(span_danger("Infused with slime jelly, the core begins to expand uncontrollably!"))
 	S.icon_state = "[S.base_state]_active"
 	S.active = TRUE
-	addtimer(CALLBACK(S, /obj/item/grenade.proc/prime), rand(15,60))
+	var/obj/item/slime_extract/M = holder.my_atom
+	addtimer(CALLBACK(S, /obj/item/grenade.proc/prime), rand((M.react_time - 3),(M.react_time + 2)))
```

Comment:
This results in rand(47, 52) not rand(15,60)

Reply:
Zamn